hasher-matcher-actioner/webapp/src/settings/ActionRuleSettingsTab.jsx

```diff
@@ -41,8 +41,45 @@ const actions = [
   },
 ];
 
+function actionRuleNameIsUnique(name, actionrules) {
+  if (name) {
+    const nameLower = name.toLowerCase();
+    return (
+      actionrules.findIndex(
+        actionRule => actionRule.name.toLowerCase() === nameLower,
+      ) === -1
+    );
+  }
+
+  return true;
+}
+
+function actionRuleIsValid(actionRule, actionrules) {
+  return (
+    actionRule.name &&
+    actionRule.must_have_labels &&
```

Comment:
So we cant have a rule with only must_not_have_labels?

Reply:
That's right. I made that decision for now... I couldn't envision blindly accepting content to action on without some kind of "positive" classification criteria. Easy to revisit later, if we find a driving use case.